test/extended/oauth/http1.go

```diff
@@ -12,7 +12,7 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var _ = g.Describe("[sig-auth][Feature:OAuthServer] OAuth server", func() {
+var _ = g.Describe("[sig-auth][Feature:OAuthServer] OAuth server [apigroup:auth.openshift.io]", func() {
```

Comment:
Is the requirement coming from `oc get --raw /.well-known/oauth-authorization-server`?